tfhe/src/shortint/server_key/mod.rs

```diff
@@ -1698,3 +1702,107 @@ where
         degree: Degree::new(max_value as usize),
     }
 }
+
+#[derive(Copy, Clone)]
+pub struct PBSConformanceParameters {
+    pub in_lwe_dimension: LweDimension,
+    pub out_glwe_dimension: GlweDimension,
+    pub out_polynomial_size: PolynomialSize,
```

Comment:
With the `in_` and `out_` prefix I think you are making an assumption on the pbs order, maybe you should leave it as `lwe_dimension` and `glwe_dimension`

Reply:
These parameters are for only for a PBS (with no KS) so there is no notion of AP or PBS order here